src/simulator/simulator.cpp

```diff
@@ -278,12 +388,21 @@ void Simulator::initSimulatorVizMarkers() {
   scale.x = CONFIG_car_length;
   scale.y = CONFIG_car_width;
   scale.z = CONFIG_car_height;
-  color[0] = 94.0 / 255.0;
-  color[1] = 156.0 / 255.0;
-  color[2] = 255.0 / 255.0;
-  color[3] = 0.8;
-  initVizMarker(robotPosMarker, "robot_position", 1, "cube", p, scale, 0.0,
-      color);
+  color[0] = 0;
+  color[1] = 0;
+  color[2] = 0;
+  color[3] = 0.8; // luminosity
+
+  for(int i = 0; i < robot_number_; i++){
+    robotPosMarkers.push_back(visualization_msgs::Marker());
+    color[0] = CONFIG_robot_colors[i].x()/255.0;
+    color[1] = CONFIG_robot_colors[i].y()/255.0;
+    color[2] = CONFIG_robot_colors[i].z()/255.0;
+
+    initVizMarker(robotPosMarkers[i],
+                  "car" + std::to_string(i) + "_robot_position", i + 2, "cube",
```

Comment:
Should be more general than "car" everywhere. Use robot, or bot, not car, and use it everywhere.

Reply:
done! Changed everything from car to robot.